CONTRIBUTING.md

```diff
@@ -0,0 +1,211 @@
+# Contributing to dosbox-staging
+
+Thank you for your interest in contributing to dosbox-staging! There are many
+ways to contribute, and we appreciate all of them. This document is a bit long,
+so here's links to the major sections:
+
+- [1 Feature requests and bug reports](#feature-requests-and-bug-reports)
+- [2 Find something to work on](#find-something-to-work-on)
```

Comment:
I like the **"Find something to work on"** chapter - and I think this could act as perfect introduction to my comment above with the major categories.  Basically, you introduce all the ways people can help, the skills that might be needed, and then present them with those four main pathways to go down.

The "Feature requests and bug reports" content can be used in the introduction bodies of most of the major sections (ie: someone who wants to do support could answer frequent questions posted in the issues, and then start writing a FAQ or wiki page).